shared-npm-packages/README.md

```diff
@@ -0,0 +1,3 @@
+# Shared Npm packages
```

Comment:
We may want to add steps for how to publish / how to get permissions to publish / telling people that publishing is now required. I think the # of people who can publish ATM is low.

Reply:
I added a note telling people to reach out to me/our team. I'd very much like to fully automate that so that it happens on CI, but this should work in the meantime.